dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/AnalyticsIndexHelper.java

```diff
@@ -160,11 +165,32 @@ private static void maybeAddTextLowerIndex(
     }
   }
 
+  private static void maybeAddDateSortOrderIndex(
+      List<Index> indexes,
+      String indexName,
+      String tableName,
+      AnalyticsTableColumn column,
+      List<String> indexColumns) {
+
+    boolean isSingleColumn = indexColumns.size() == 1;
+
+    if (column.getDataType() == TIMESTAMP && !column.isDynamicColumn() && isSingleColumn) {
```

Comment:
We could be more explicit by using `column.isStatic()` instead of `!column.isDynamicColumn()`

Reply:
done